salt/daemons/masterapi.py

```diff
@@ -836,6 +836,46 @@ def minion_runner(self, load):
         runner = salt.runner.Runner(opts)
         return runner.run()
 
+    def minion_wheel(self, load):
+        '''
+        Execute a wheel from a minion, return the wheel's function data
+        '''
+        if 'peer_wheel' not in self.opts:
+            return {}
+        if not isinstance(self.opts['peer_wheel'], dict):
+            return {}
+        if any(key not in load for key in ('fun', 'arg', 'id')):
+            return {}
```

Comment:
@gtmanfred can we [avoid multiple returns](http://docs.python-guide.org/en/latest/writing/style/#returning-values), sticking to [SESE](https://en.wikipedia.org/wiki/Single-entry_single-exit) instead? Because other starts copying this and then the code isn't fun to maintain.